php/pantheon/checks/general.php

```diff
@@ -180,4 +181,47 @@ public function message(Messenger $messenger) {
 		}
 		$messenger->addMessage(get_object_vars($this));
 	}
+
+	public function checkCoreUpdates() {
+
+		ob_start();
+		\WP_CLI::run_command(array(
+			'core',
+			'check-update'
+		), array('format' => 'json'));
+		$ret = ob_get_clean();
```

Comment:
> Without those, the json itself is printed to the output. I don't we want that printed.

Right — use `WP_CLI::runcommand()`, not `WP_CLI::run_command()`.

The former is newer and better :) See http://wp-cli.org/docs/internal-api/wp-cli-runcommand/